app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GreetingsController < ApplicationController
+  def index; end
```

Comment:
Make sure to create `Greeting.js` component inside your javascript folder which get render in `App.js` component. I see you import `import Greeting from "./Greeting";` but you don't have that component inside your `components` folder.

One more thing you have to have to create `redux` folder which consists of a `store`, an `action` , and a `reducer` that uses the url to the endpoint from the Rails API. Implement all these changes inside your `app/javascript` folder.